doc/decisions/0005-use-consistent-value-references-in-validation.md

```diff
@@ -0,0 +1,95 @@
+# 5. Use consistent value references in validation
+
+## Context
+
+We reference dynamic values (metadata, answers) in placeholders, validation, and routing. The agreed structure for referencing dynamic values is:
+
+```
+{
+    “source”: “”,
+    “identifier”: “”
+}
+```
+
+This structure is used by placeholders and is part of the agreed [changes to routing](https://github.com/ONSdigital/eq-schema-validator/blob/v3/doc/decisions/0004-extend-and-improve-routing-rules.md). Validation is inconsistent with these.
+
+## Proposal
+
+Update validation so that it uses the same consistent way of referencing dynamic values as placeholders and routing. This will be changed for minumum and maximum validation on numbers and dates (all current validaiton we support). 
```

Comment:
```suggestion
Update validation so that it uses the same consistent way of referencing dynamic values as placeholders and routing. This will be changed for minimum and maximum validation on numbers and dates (all current validation we support). 
```